app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -75,4 +79,12 @@ def textual_refresh_status
       :title => @ems.last_refresh_error
     }
   end
+
+  def textual_topology
+    label = 'Topology'
+    h     = {:label => label, :image => "container_topology"}
+    h[:link]  = url_for(:controller => 'container_topology', :action => 'show', :id => @ems.id)
+    h[:title] = "Show topology"
+    h
```

Comment:
It should probably say:

    def textual_topology
        {:label => N_('Topology'),
         :image => 'container_topology',
         :link  = url_for(:controller => 'container_topology', :action => 'show', :id => @ems.id),
         :title => N_("Show topology")}
    end